repos/fdbt-site/src/pages/products/exports.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { ReactElement } from 'react';
+import { NextPageContextWithSession } from '../../interfaces';
+import { BaseLayout } from '../../layout/Layout';
+import { exportEnabled } from '../../constants/featureFlag';
+import { getAndValidateNoc, getCsrfToken } from '../../utils';
+import { getS3FolderCount, getS3Exports, retrieveAndZipExportedNetexForNoc, getNetexSignedUrl } from '../../data/s3';
+import { redirectTo } from '../../utils/apiUtils';
+import { MATCHING_DATA_BUCKET_NAME, NETEX_BUCKET_NAME } from '../../constants';
+import CsrfForm from '../../components/CsrfForm';
+
+const title = 'Exports';
+const description = 'View and access your settings in one place.';
+
+interface GlobalSettingsProps {
+    counts: { matchingDataCount: number; name: string; netexCount: number; signedUrl: string | null }[];
+    csrf: string;
+}
+
+const Exports = ({ counts, csrf }: GlobalSettingsProps): ReactElement => {
+    return (
+        <>
+            <BaseLayout title={title} description={description} showNavigation>
+                <div className="govuk-grid-row">
+                    <div className="govuk-grid-column-three-quarters">
+                        <h1 className="govuk-heading-xl">Exports</h1>
+
+                        <CsrfForm csrfToken={csrf} method={'post'} action={'/api/exports'}>
+                            <button type="submit" className="govuk-button">
+                                Export all
+                            </button>
+                        </CsrfForm>
+
+                        <table className="govuk-table">
+                            <thead className="govuk-table__head">
+                                <tr className="govuk-table__row">
+                                    <th scope="col" className="govuk-table__header">
+                                        Export Name
+                                    </th>
+                                    <th scope="col" className="govuk-table__header">
+                                        Files
+                                    </th>
+                                    <th scope="col" className="govuk-table__header">
+                                        Status
+                                    </th>
+                                </tr>
+                            </thead>
+                            <tbody className="govuk-table__body">
+                                {counts.map((it) => (
```

Comment:
Could we change 'it' to 'count'? Is counts + count the best name for these, also?

Reply:
Renamed to exports, and exportDetails